.sonarcloud.properties

```diff
@@ -13,4 +13,4 @@ sonar.sources=.
 sonar.exclusions=**/Makefile, **/migrations/**, **bc_obps/registration1/tests/**, **/bciers/apps/registration1/tests/**, **/bciers/apps/registration1/e2e/**
 
 # Ignore duplication scanning for registration part 2 (will be removed once part 2 is merged into part 1)
-sonar.cpd.exclusions=**/bciers/apps/registration/**/*
+sonar.cpd.exclusions=**/bciers/apps/registration/**/*, **/bciers/apps/registration1/**/*
```

Comment:
Had to add duplication exclusion for registration1, since I moved the original Layout components back to it so I could refactor them without worrying about changes/regressions.